lua/nvim_lsp/vimls.lua

```diff
@@ -0,0 +1,69 @@
+local skeleton = require "nvim_lsp/skeleton"
+local util = require "nvim_lsp/util"
+local lsp = vim.lsp
+local api = vim.api
+
+local server_name = "vimls"
+local bin_name = "vim-language-server"
+
+local installer =
+    util.npm_installer {
+    server_name = server_name,
+    packages = {"vim-language-server"},
+    binaries = {bin_name}
+}
+
+skeleton[server_name] = {
+    default_config = {
+        cmd = {bin_name, "--stdio"},
+        filetypes = {"vim"},
+        root_dir = function(fname)
+          return util.find_git_ancestor(fname) or vim.loop.os_homedir()
+        end;
+        log_level = lsp.protocol.MessageType.Warning,
+        settings = {},
+        init_options = {
+            iskeyword = "@,48-57,_,192-255,-#",
+            vimruntime = "",
+            runtimepath = "",
+            diagnostic = {
+                enable = true
+            },
+            indexes = {
+                runtimepath = true,
+                gap = 100,
+                count = 3,
+                projectRootPatterns = {"nvim", ".git", "autoload", "plugin"}
```

Comment:
"runtime" would be useful too I suppose (first in the list)

Reply:
Fixed